src/Agent.php

```diff
@@ -39,6 +39,7 @@ class Agent extends Mobile_Detect {
     protected static $additionalBrowsers = array(
         'Opera'             => 'Opera|OPR',
         'Edge'              => 'Edge',
+        'Vivaldi'           => 'Vivaldi', //Must be above Chrome, otherwise it'll fail
```

Comment:
Can you delete this comment? Might be better to put a general comment above this variable because this applies to other entries as well.